src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -19,18 +19,46 @@ If you're new to the topic of nodes, we recommend first checking out our user-fr
 
 ## What are nodes and clients? {#what-are-nodes-and-clients}
 
-A "node" is a computer running Ethereum client software. A client is an implementation of Ethereum that verifies all transactions in each block, keeping the network secure and the data accurate. Up until The Merge, a single piece of software is required to run a full node (or two to run a mining node). After The Merge, two pieces of client software are required to run a full node (three to run a validator node), one to handle and gossip transactions (execution client), one to handle block gossip and fork choice (consensus client) and an optional validator client that handles block production and "voting" on blocks received from its peers.
+"Node" refers to a running piece of client software. A client is an implementation of Ethereum that verifies all transactions in each block, keeping the network secure and the data accurate.
 
-You can see a real-time view of the Ethereum network by looking at this [map of nodes](https://etherscan.io/nodetracker).
+An Ethereum node consists of two pieces of software: execution client and consensus client. 
```

Comment:
I followed "consensus client, execution client" terminology used in current docs but I agree it can be confusing. I will add info on various terminology to give better context